docs/services/gracc-corrections.md

```diff
@@ -0,0 +1,62 @@
+Installing GRACC Corrections
+============================
+
+GRACC Corrections are used to modify records during the summarization process.  RAW records are not modified in the correction process.  The correction is applied after summarization and aggregation, but before the record is enriched with data from [Topology](https://topology.opensciencegrid.org/).
+
+The correction is step 3 in the GRACC summary record workflow:
+
+1. Raw record is received.  The raw record is never modified
+2. Summarizer aggregates the raw records
+3. Corrections are applied
+4. Summarized records are enriched by Topology
+5. Summarized and enriched records are uploaded to GRACC
+
+We can currently correct:
+
+* VO Names
+* Project Names
+* OIM_Site (using the `Host_description` field)
+
+Limitations
+-----------
+
+Additional corrections can be written, but some attributes are protected.  GRACC uses specific attributes to detect duplicate records.  Protected records for summarization are:
```

Comment:
this kind of makes sense after you read the following example, but upon first read, these three sentences sounded kind of disconnected to me.

> Additional corrections can be written, but some attributes are protected.

What is a "protected" attribute?  Is that an attribute for which a correction can't be written?  (I think that's what you are trying to say, but to me the implication is a bit unclear.)

> GRACC uses specific attributes to detect duplicate records.

Are these attributes for detecting duplicate records the same as the protected attributes, or is this a second additional statement, separate from protected attributes?

>Protected records for summarization are:

Again, not clear if these are for the duplicates or something else.  Also, now you are calling them protected "records" instead of attributes.

... anyway like i said i think it makes sense after reading the example later, but this part was a little confusing for me at first pass.  Maybe it can be reworded.